java/11/Dockerfile

```diff
@@ -5,9 +5,9 @@ LABEL       author="Michael Parker" maintainer="parker@pterodactyl.io"
 LABEL       org.opencontainers.image.source="https://github.com/pterodactyl/yolks"
 LABEL       org.opencontainers.image.licenses=MIT
 
-RUN 		apt-get update -y \
- 			&& apt-get install -y curl ca-certificates openssl git tar sqlite3 fontconfig tzdata iproute2 \
- 			&& useradd -d /home/container -m container
+RUN 		    apt update -y \
```

Comment:
This is a mix of tabs and spaces. These spaces are unnecessary and offset the whole line compared to the rest making it inconsistent.  Remove the space to keep this consistent

Reply:
The mix was before also :) Now i romoed spaces and only use tabs